common/client/mock_hashable_test.go

```diff
@@ -0,0 +1,22 @@
+package client
+
+// Hashable - simple implementation of types.Hashable interface to be used as concrete type in tests
+type Hashable string
+
+func (h Hashable) Cmp(c Hashable) int {
+	if h == c {
+		return 0
+	} else if h > c {
+		return 1
+	}
+
+	return -1
```

Comment:
There is a std lib func for `Ordered` types:
```suggestion
	return cmp.Compare(h, c)
}
```